app/controllers/posts_controller.rb

```diff
@@ -7,4 +7,21 @@ def index
   def show
     @post = Post.find_by(id: params[:id], author_id: params[:user_id])
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
+    @post.author = current_user
+    @post.comments_counter = 0
+    @post.likes_counter = 0
+    redirect_to user_posts_path(current_user.id) if @post.save
+    render plain: 'Error occured while saving data'
```

Comment:
[ ] - Nice working on your Post controller, I get the error in the screenshot when trying to create a post. You need to remove the `render plain: 'Error occured while saving data`.
![image](https://user-images.githubusercontent.com/48371156/189498882-3fa09aa3-1bc5-4704-9aae-0711ef0fed62.png)